### DISCUSSION TITLE
Can someone help?

### DISCUSSION THREAD

**Discussion by Mesut70** [microsoft/vscode-discussions#821] [Extension Development QnA] [answered]
Hi all,

Im sorry if i am on a wrong place but i am a newbie.
I have a question about my script (telegram bot).

When i write this 

const bot = new Telegrambot(token, {polling: true});
bot.on('message', function(msg) {console.log('Received a message');
console.log (msg)
const res = 'Hi (first_name) I received your message: ${msg.text}!';
bot.sendMessage(msg.chat.id,  res);
})  

I get the message but with the first name and msg.text real hard as text so i get

Hi (first_name) instead of Hi Johan or Hi Patrick the reall first name.

How can i fix this?

Thanks a lot!

**Comment by gjsjohnmurray** [comment, marked as answer]
Yes, this is not the right place for general coding questions. It is currently focused on developers creating VS Code extensions.

I suggest you read https://en.wikipedia.org/wiki/String_interpolation and make sure you use the correct syntax for the language you are coding in. For example in Javascript and Typescript you must use backquotes.

**Comment by olguzzar** [comment]
Closing as this is off-topic for the forum, which is focused on extension development.
